orchagent/switchorch.cpp

```diff
@@ -0,0 +1,85 @@
+#include <map>
+
+#include "switchorch.h"
+
+using namespace std;
+using namespace swss;
+
+extern sai_object_id_t gSwitchId;
```

Comment:
since we are going to have SwitchOrch, I suggest to replace global gSwitchId variable to 
static member + some static getSwitchId() method in SwitchOrch class

Reply:
I agree. I would suggest to move it to a separate pull request and I'm thinking about creating a Switch class to move all the Orch instances into this class as singletons. What do you think about this idea?

Reply:
sure